README.md

```diff
@@ -9,6 +9,14 @@ A REST API designed to interact with the Open Food Facts _Open Prices_ database.
 - Frontend repo: https://github.com/openfoodfacts/open-prices-frontend
 - Documentation: https://openfoodfacts.github.io/open-prices/
 
+## 🎨 Design & User interface
+- The Open Prices server is exposed through the Mobile App and the Open Prices frontend (and probably elsewhere in the future, thanks to web components)
+- We strive to thoughfully design every feature before we move on to implementation, so that we respect Open Food Facts' graphic charter and nascent design system, while having efficient user flows.
```

Comment:
```suggestion
- We strive to thoughtfully design every feature before we move on to implementation, so that we respect Open Food Facts' graphic charter and nascent design system, while having efficient user flows.
```